token-list.json

```diff
@@ -40106,6 +40106,38 @@
       "name": "Wrapped Bitcoin",
       "symbol": "WBTC"
     },
+    {
+      "address": "0x21F86113c57061bd5c5ee8953322b712b7911e11",
+      "chainId": 80001,
+      "decimals": 18,
+      "logoURI": "https://raw.githubusercontent.com/atomica-dev/custom-token-list/master/icons/0x21F86113c57061bd5c5ee8953322b712b7911e11.png",
```

Comment:
I suggest to have the only icon for tokens without logo.

Reply:
It means in case token has no logo, logoURI should be empty and UI must know how to handle its icon.